projects/ad2s1210_iio/app/app_config_mbed.h

```diff
@@ -0,0 +1,59 @@
+/***************************************************************************//**
+ *   @file    app_config_mbed.h
+ *   @brief   Header file for Mbed platform configurations
+********************************************************************************
+ * Copyright (c) 2023 Analog Devices, Inc.
+ * Copyright (c) 2023 BayLibre, SAS.
+ * All rights reserved.
+ *
+ * This software is proprietary to Analog Devices, Inc. and its licensors.
+ * By using this software you agree to the terms of the associated
+ * Analog Devices Software License Agreement.
+*******************************************************************************/
+
+#ifndef APP_CONFIG_MBED_H_
+#define APP_CONFIG_MBED_H_
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include <stdint.h>
+#include <PinNames.h>
+
+#include "mbed_uart.h"
+#include "mbed_irq.h"
+#include "mbed_gpio_irq.h"
+#include "mbed_spi.h"
+#include "mbed_pwm.h"
+#include "mbed_gpio.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definition ***********************/
+/******************************************************************************/
+#define SPI_CSB		ARDUINO_UNO_D10
+#define SPI_HOST_SDO	ARDUINO_UNO_D11
+#define SPI_HOST_SDI	ARDUINO_UNO_D12
+#define SPI_SCK		ARDUINO_UNO_D13
+#define PWM_TRIGGER     ARDUINO_UNO_D3
+
+/* UART Common Pin Mapping on SDP-K1 */
+#define UART_TX		CONSOLE_TX
+#define	UART_RX		CONSOLE_RX
+
+#define SAMPLING_RATE                                   (16000)
```

Comment:
Is this the maximum sampling rate mentioned in the datasheet?
if not, a short comment can be added stating the reason why this is only 16ksps.

Example - https://github.com/analogdevicesinc/precision-converters-firmware/blob/main/projects/ad4696_iio/app/app_config_mbed.h#L50

Reply:
there is not max sampling rate mentioned in datasheet. i kind out find out a sane value max experimentally from my current setup with flywires + the breakout board. not sure exactly what the max value i  can use is.  
ill add a comment on how this value was experimentally  selected.